portfolio/src/main/java/com/google/sps/data/TimeRangeGroup.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+import java.util.Collection;
+
+public interface TimeRangeGroup {
+
+  /**
+   * Merges the collection of time ranges that is passed into the constructor so that all time
+   * ranges are disjoint.
+   */
+  public Collection<TimeRange> mergeInputTimeRanges(Collection<TimeRange> timeRanges);
+
+  /** Adds a new time range into the collection of time ranges. */
+  public void addTimeRange(TimeRange timeRange);
+
+  /** Returns the collection of all time ranges. */
+  public Collection<TimeRange> getAllTimeRanges();
+
+  /** Checks if a time range exists in the collection. */
```

Comment:
Consider clarifying the behavior. For example, if the time range group has 1pm-2pm and 3pm-4pm, and this method is called with 3pm-3:30pm, does this method return true? (I think it should.)

Reply:
Updated.